docs/authentication/social-connections/microsoft.mdx

```diff
@@ -75,3 +75,40 @@ This can be done by navigating to the Authentication section and checking the op
 Congratulations! Social connection with Microsoft Azure is now configured for your instance.
 
 Stay tuned for updates as Clerk integrates support for more Microsoft social connection options.
+
+### Stay secure against the nOAuth vulnerability
+
+A vulnerability called [nOAuth](https://www.descope.com/blog/post/noauth) has been recently discovered in Microsoft's Azure AD OAuth applications. This vulnerability can lead to full account takeovers, as attackers can spoof the provided email and potentially gain full control of a victim's account.
+
+To protect users, Clerk has implemented stringent measures against the Microsoft OAuth provider, enforcing stricter checks on verified email addresses. However, this has impacted the functionality of users using Azure AD.
+
+Microsoft has introduced an optional claim, `xms_edov`, that can be added to your Azure application. This claim provides additional context to determine whether the returned email is verified.
+
+To enable this optional claim, you'll need to follow these steps:
+
+Navigate to your Azure application's **Token configuration** page, and click on **Add optional claim**.
+
+![Add optional claims](/docs/images/authentication-providers/microsoft/noauth-optional-claims-init.png)
+
+Select the `email` and `xms_pdl` claims for both **ID** and **Access**, and click on **Add**.
+
+<Callout type="info">
+    At the time of writing, the `xms_edov` claim is still in preview, and may not be available for all applications. So
+    we'll choose another one from the list and we'll rename it later on in the manifest.

```

Comment:
```suggestion
    At the time of writing, the `xms_edov` claim is still in preview, and may not be available for all applications. So we'll choose another one from the list and we'll rename it later on in the manifest.

```

Reply:
Hmm, looks like IDE splits the lines on format because it parses this as a component. Anyway, it's fixed – thanks!